### ISSUE TITLE
`terraform get` crashes when module has invalid export syntax

### ISSUE THREAD

**Issue by tkellen** [hashicorp/terraform#4733]
Reproduction:

**File tree:**
```
.
├── module
│   └── main.tf
└── terraform.tf

1 directory, 2 files
```

**module/main.tf**
```
exports = "test" { value = "test" }
```

**terraform.tf**
```
module "module" {
  source = "./module"
}
```

With this configuration, run `terraform get`.

**Comment by tkellen** [comment]
Oh, and I'm running terraform 0.6.8. (upgrading to terraform 0.6.9 now to see if that fixes this...)

**Comment by jen20** [comment]
Hi @tkellen! I just tried reproducing this using the `master` branch of Terraform, and get the following output:

```
 ~/Code/terraform/repros/4733/1 $  terraform get
Get: file:///Users/James/Code/terraform/repros/4733/1/module
 ~/Code/terraform/repros/4733/1 $  terraform plan
There are warnings and/or errors related to your configuration. Please
fix these before continuing.

Errors:

  * 1 error(s) occurred:

* module module.root: 1 error(s) occurred:

* Unknown root level key: exports
```

This is pretty much what I'd expect to see in this situation, so it either looks like this has been fixed already, or there is something environment specific. Optimistically (!) I'm going to close this issue as "fixed on `master`" - if you can still reproduce it could you please include the output of `terraform get`, as well as information about what platform you are on and I'll try again to reproduce? Thanks!

**Comment by tkellen** [comment]
Confirmed fixed on 0.6.9!

**Comment by tkellen** [comment]
thanks @jen20
